### PR TITLE
updated stafftime view and model

### PR DESCRIPTION
**Pull request by kate-mills**
I made a small update to our stafftime view to test a pull request.

### CODE REVIEW

File: room/templates/stafftime_static.html [hunk 1]
```diff
@@ -65,7 +65,9 @@
     Description
   </td>
   <td>
-    {{ entity.description|escape|replace('\n', '<br>\n') }}
```
Comment (babybunny) [inline room/templates/stafftime_static.html]: We do have to worry about HTML escaping in these templates.  For example what if someone enters a description like "I am full of mischief <script> ... do something terrible .."

See this about the autoescaping in jinja2.   
 http://jinja.pocoo.org/docs/2.9/api/#autoescaping

I'm not sure if it's enabled.  It probably should be.

**Review by babybunny — CHANGES_REQUESTED** [review]
Do you have a demo version?  It's a good practice to create one and share the URL so I (as reviewer) can see the effect of the changes.

**Comment by kate-mills** [comment]
I created a demo version at https://kate-testing-dot-rebuildingtogethercaptain-hrd.appspot.com

**Review by babybunny — APPROVED** [review]
Looks good!

**Pull request merged**